src/routes/utils.rs

```diff
@@ -0,0 +1,121 @@
+use crate::models::service::utils::{
```

Comment:
I find this file name very confusing, as I would expect utility methods here

Reply:
that's my understanding of the gas estimation and a data decoder endpoint. How would you classify these endpoints?

Reply:
Ibmean these are endpoint methods which is something else for me than utils :D

The estimation endpoint is a safe related endpoint.

The master copies could be part of the about.

And data decoding could be a separate module ... Not sure about this one.